docker-compose.yml

```diff
@@ -34,3 +44,11 @@ elasticsearch:
 
 redis:
   image: redis
+
+rabbitmq:
+  image: rabbitmq:3-management
+  hostname: olympia
+  name: olympia
```

Comment:
I was working under the assumption that these keys just get passed to the CLI and olympia expects the rabbitmq to be called olympia. I can try removing them and see if they actually make a difference.